### ISSUE TITLE
Add "blocked" label to issues / PRs?

### ISSUE THREAD

**Issue by jackfirth** [actions/labeler#15]
Is there a standard action like this one, but which adds labels based on a pull request's description (or an issue's description) instead of its changed files? I'd like to automatically add a `blocked` label to any issue or pull request whose description includes text like "blocked on #123".

**Comment by damccorm** [comment]
Not yet. https://github.com/bryanmacfarlane/tag-ur-it is a bot that does something similar to this that we probably want to port to actions at some point.

cc @bryanmacfarlane

**Comment by MaksimZhukov** [comment]
Hello everyone! 
I am closing this issue as a duplicate of [this one](https://github.com/actions/labeler/issues/55). Please monitor the feature status in scope of that issue.
